src/vivarium_testing_utils/automated_validation/interface.py

```diff
@@ -315,3 +317,20 @@ def _get_age_groups(self) -> pd.DataFrame:
             # relabel index level age_group_name to age_group
 
         return age_groups.rename_axis(index={"age_group_name": "age_group"})
+
+    def cache_gbd_data(
+        self, data_key: str, data: pd.DataFrame | pd.Series[float], overwrite: bool = False
+    ) -> None:
+        """Upload a custom DataFrame or Series to the context given by a data key."""
+        if isinstance(data, pd.Series):
+            data = data.to_frame(name="value")
+        mapped_data: pd.DataFrame = self._format_to_vivarium_inputs_conventions(data)
+        self.data_loader.cache_gbd_data(data_key, mapped_data, overwrite=overwrite)
+
+    def _format_to_vivarium_inputs_conventions(self, data: pd.DataFrame) -> pd.DataFrame:
+        """Format a DataFrame or Series to vivarium inputs conventions."""
+        data = vi.scrub_gbd_conventions(data, self.location)
+        data = vi.split_interval(data, interval_column="age", split_column_prefix="age")
+        data = vi.split_interval(data, interval_column="year", split_column_prefix="year")
+        formatted_data: pd.DataFrame = vi.sort_hierarchical_data(data)
```

Comment:
@rmudambi I would like to talk about this. If data is pulled via get_draws (that is going away and we are unsure what that will look like in the future) there are extra columns that are not being address here. I thought about taking the data_key and parsing to try and leverage validation.validate_for_simulation called from within load_standard_data but that would be pretty involved for this ticket.

Reply:
To be clear, `get_draws()` is not going away, it's just being deprecated, and we will deal with that when the time comes. What extra columns are you worried about?